server/monitor/src/test/java/org/apache/accumulo/monitor/view/WebViewsTest.java

```diff
@@ -130,6 +130,7 @@ public void testGetTracesSummaryValidationConstraint() {
   /**
    * Silly stub to handle MessageBodyWriter for Hashmap. Registered in configure method and auto-wired by Jersey.
    */
+  @SuppressWarnings("rawtypes")
```

Comment:
`<?,?>` as the generic type would be much preferred over suppressing the warning. It's always better to be explicit, and I'm not a fan of instructing the compiler to ignore warnings when there's an easy fix.

Also, I just noticed that this test creates an external web service to test against. As such, this test should be renamed to be `WebViewsIT`, because it's an integration test, and not just a unit test.

And, don't worry about the conflicts with master. I can take care of that for you (going to rebase before merge). I can also address these last few minor issues, too, during the rebase/merge process.

Reply:
RE: IntegrationTests which @Category class should this annotation point at?

Reply:
Ugh, I forgot about categories. Don't worry about that. I'll deal with it as a follow-on issue. It's fine as it is for now.